tests/unit/modules/test_zypperpkg.py

```diff
@@ -564,17 +569,45 @@ def test_upgrade_success(self):
                             "Dummy",
                             "--from",
                             "Dummy2",
-                        )
-                        zypper_mock.assert_any_call(
-                            "dist-upgrade",
-                            "--auto-agree-with-licenses",
-                            "--dry-run",
-                            "--from",
-                            "Dummy",
-                            "--from",
-                            "Dummy2",
-                            "--debug-solver",
-                        )
+                        
```

Comment:
```suggestion
                        )
```

A closing bracket is missing here.